### PR TITLE
change password UI and APi integration

### PR DESCRIPTION
**Pull request by Deeptrentium**
Screenshot_20240917-115605

### CODE REVIEW

File: app/(protected)/change-password/changePassword.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          password_confirmation: values?.confirmPassword,
+        }),
+      ).unwrap();
+      toast.show(response?.message, {
+        type: ToastType.Custom,
+        data: {
+          type: ToastTypeProps.Success,
+        },
+      });
+      router.navigate('/(protected)/more-menu/moreMenu');
```
Comment (sagark1510) [inline app/(protected)/change-password/changePassword.tsx:38]: what is the logic behind having more-menu/moreMenu route?

File: app/(protected)/more-menu/moreMenu.tsx [hunk 2]
```diff
@@ -82,6 +83,8 @@ const MoreMenu = () => {
     {
       label: 'changePassword',
       icon: <LockIcon />,
+      onPress: () =>
+        router.navigate('/(protected)/change-password/changePassword'),
```
Comment (sagark1510) [inline app/(protected)/more-menu/moreMenu.tsx:87]: same as above

File: components/organisms/ChangePasswordForm/ChangePasswordForm.tsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+            right
+            secureTextEntry={secureConfirmTextEntry}
+          />
+        </KeyboardAwareScrollView>
+      </Flexed>
+      <BottomContainer>
+        <ButtonSubmit
+          onPress={handleChangePassword}
+          loading={loading}
+          valid={valid}>
```
Comment (sagark1510) [inline components/organisms/ChangePasswordForm/ChangePasswordForm.tsx:120]: button van not be valid or invalid.. pass variant

**Review by sagark1510 — CHANGES_REQUESTED** [review]

File: app/(protected)/change-password/changePassword.tsx (hunk 1 above)
Comment (Deeptrentium) [reply app/(protected)/change-password/changePassword.tsx:38]: its more menu screen so once password changes its will be redirect to more menu screen

File: app/(protected)/more-menu/moreMenu.tsx (hunk 2 above)
Comment (Deeptrentium) [reply app/(protected)/more-menu/moreMenu.tsx:87]: I just follow the structure that currently we have

**Review by sagark1510 — APPROVED** [review]

**Pull request merged**